pom.xml

```diff
@@ -69,6 +69,11 @@
       <version>1.11-SNAPSHOT</version>
     </dependency>
     <dependency>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>pipeline-github-lib</artifactId>
+      <version>1.0-SNAPSHOT</version>
```

Comment:
There is no “creation PR” there, so leaving feedback here:

That plugin does not really need a dep on `github-branch-source`. It only actually uses `git`, at least currently.